runners/spark/src/main/java/org/apache/beam/runners/spark/translation/streaming/StreamingTransformTranslator.java

```diff
@@ -530,6 +530,38 @@ public String toNativeString() {
     };
   }
 
+  private static <K, V, W extends BoundedWindow> TransformEvaluator<Reshuffle.Keys<K, V>> reshuffleKeys() {
```

Comment:
As well as here.

Reply:
Updated and moved duplicate code in shared method/util.